en/starter/generator.md

```diff
@@ -10,7 +10,7 @@ redirect_from: "/starter/generator.html"
 
 Use the application generator tool, `express-generator`, to quickly create an application skeleton.
 
-Install `express-generator` with the following command:
+Install `express` with the following command:
```

Comment:
> Install `express` with the following command

Yes, definitely very poor wording here. It sounds like you're _installing express_, which to me, sounds like this module express. I understand that there are people (like me), who would think `express` means the module and other (yourself) who would think `express` means the command line program.

I'm not saying either is right or wrong, but as you can see, with both thinking those different things, I think it means the entire wording just needs to be completely redone to make the intent clear.